### ISSUE TITLE
update failed for AnAction with ID=SymfonyWebDeploymentDownload: SERVER_CONFIG

### ISSUE THREAD

**Issue by Scifa** [Haehnchen/idea-php-symfony2-plugin#833]
Since updating phpStorm on mac plugin keeps failing.

`update failed for AnAction with ID=SymfonyWebDeploymentDownload: SERVER_CONFIG
java.lang.NoSuchFieldError: SERVER_CONFIG
	at fr.adrienbrault.idea.symfony2plugin.webDeployment.actions.SymfonyWebDeploymentDownloadAction.update(SymfonyWebDeploymentDownloadAction.java:31)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:122)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:188)
	at com.intellij.openapi.actionSystem.impl.Utils.updateGroupChild(Utils.java:265)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:239)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:209)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:147)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:86)
	at com.intellij.openapi.actionSystem.impl.Utils.fillMenu(Utils.java:280)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.a(ActionMenu.java:324)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.access$600(ActionMenu.java:51)
	at com.intellij.openapi.actionSystem.impl.ActionMenu$MenuListenerImpl.menuSelected(ActionMenu.java:278)
	at javax.swing.JMenu.fireMenuSelected(JMenu.java:1038)
	at javax.swing.JMenu$MenuChangeListener.stateChanged(JMenu.java:1118)
	at javax.swing.DefaultButtonModel.fireStateChanged(DefaultButtonModel.java:349)
	at javax.swing.DefaultButtonModel.setSelected(DefaultButtonModel.java:232)
	at javax.swing.JMenu.setSelected(JMenu.java:305)
	at com.apple.laf.ScreenMenu$2.run(ScreenMenu.java:135)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:301)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:90)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.awt.EventQueue$4.run(EventQueue.java:729)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at com.intellij.ide.IdeEventQueue.d(IdeEventQueue.java:843)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:679)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:391)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)`

**Comment by stof** [comment]
I'm getting the same error

**Comment by Scifa** [comment]
I think it's to do with this, so will have to wait for patch.

https://youtrack.jetbrains.com/issue/WI-34281

**Comment by Haehnchen** [comment]
to be fixed with #826

**Comment by iltar** [comment]
Just updated to the latest EAP and this is still broken so I assume it's still waiting for something?

```
PhpStorm 2017.1 EAP
Build #PS-171.1834.12, built on December 22, 2016
PhpStorm EAP User
Expiration date: January 21, 2017
JRE: 1.8.0_112-release-b620 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
```

```
update failed for AnAction with ID=SymfonyWebDeploymentDownload: SERVER_CONFIG
java.lang.NoSuchFieldError: SERVER_CONFIG
	at fr.adrienbrault.idea.symfony2plugin.webDeployment.actions.SymfonyWebDeploymentDownloadAction.update(SymfonyWebDeploymentDownloadAction.java:31)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:124)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:234)
	at com.intellij.openapi.actionSystem.impl.Utils.updateGroupChild(Utils.java:311)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:285)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:255)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:187)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:99)
	at com.intellij.openapi.actionSystem.impl.Utils.fillMenu(Utils.java:340)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.e(ActionMenu.java:326)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.access$600(ActionMenu.java:54)
	at com.intellij.openapi.actionSystem.impl.ActionMenu$MenuListenerImpl.menuSelected(ActionMenu.java:280)
	at javax.swing.JMenu.fireMenuSelected(JMenu.java:1038)
	at javax.swing.JMenu$MenuChangeListener.stateChanged(JMenu.java:1118)
	at javax.swing.DefaultButtonModel.fireStateChanged(DefaultButtonModel.java:349)
	at javax.swing.DefaultButtonModel.setSelected(DefaultButtonModel.java:232)
	at javax.swing.JMenu.setSelected(JMenu.java:305)
	at javax.swing.JMenu.menuSelectionChanged(JMenu.java:1187)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.menuSelectionChanged(ActionMenu.java:244)
	at javax.swing.MenuSelectionManager.setSelectedPath(MenuSelectionManager.java:125)
	at javax.swing.plaf.basic.BasicMenuUI$Handler.mouseEntered(BasicMenuUI.java:526)
	at java.awt.Component.processMouseEvent(Component.java:6542)
	at javax.swing.JComponent.processMouseEvent(JComponent.java:3324)
	at java.awt.Component.processEvent(Component.java:6298)
	at java.awt.Container.processEvent(Container.java:2236)
	at java.awt.Component.dispatchEventImpl(Component.java:4889)
	at java.awt.Container.dispatchEventImpl(Container.java:2294)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4888)
	at java.awt.LightweightDispatcher.retargetMouseEnterExit(Container.java:4676)
	at java.awt.LightweightDispatcher.trackMouseEnterExit(Container.java:4654)
	at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4505)
	at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4466)
	at java.awt.Container.dispatchEventImpl(Container.java:2280)
	at java.awt.Window.dispatchEventImpl(Window.java:2746)
	at java.awt.Component.dispatchEvent(Component.java:4711)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:90)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.awt.EventQueue$4.run(EventQueue.java:729)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at com.intellij.ide.IdeEventQueue.d(IdeEventQueue.java:809)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:642)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:365)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by dop3** [comment]
Same here:
```
PhpStorm 2016.3.2
Build #PS-163.10504.2, built on December 20, 2016
JRE: 1.8.0_112-release-408-b6 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
```

```
update failed for AnAction with ID=SymfonyWebDeploymentDownload: SERVER_CONFIG
java.lang.NoSuchFieldError: SERVER_CONFIG
	at fr.adrienbrault.idea.symfony2plugin.webDeployment.actions.SymfonyWebDeploymentDownloadAction.update(SymfonyWebDeploymentDownloadAction.java:31)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performDumbAwareUpdate(ActionUtil.java:122)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:188)
	at com.intellij.openapi.actionSystem.impl.Utils.updateGroupChild(Utils.java:265)
	at com.intellij.openapi.actionSystem.impl.Utils.a(Utils.java:239)
	at com.intellij.openapi.actionSystem.impl.Utils.b(Utils.java:209)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:147)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroup(Utils.java:86)
	at com.intellij.openapi.actionSystem.impl.Utils.fillMenu(Utils.java:280)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.d(ActionMenu.java:324)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.access$600(ActionMenu.java:51)
	at com.intellij.openapi.actionSystem.impl.ActionMenu$MenuListenerImpl.menuSelected(ActionMenu.java:278)
	at javax.swing.JMenu.fireMenuSelected(JMenu.java:1038)
	at javax.swing.JMenu$MenuChangeListener.stateChanged(JMenu.java:1118)
	at javax.swing.DefaultButtonModel.fireStateChanged(DefaultButtonModel.java:349)
	at javax.swing.DefaultButtonModel.setSelected(DefaultButtonModel.java:232)
	at javax.swing.JMenu.setSelected(JMenu.java:305)
	at com.apple.laf.ScreenMenu$2.run(ScreenMenu.java:135)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:301)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:90)
	at java.awt.EventQueue$4.run(EventQueue.java:731)
	at java.awt.EventQueue$4.run(EventQueue.java:729)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:728)
	at com.intellij.ide.IdeEventQueue.e(IdeEventQueue.java:843)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:679)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:391)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by Haehnchen** [comment]
fixed by 0.13.133
